src/app/zap-templates/zcl/temperature-measurement-cluster.xml

```diff
@@ -25,8 +25,8 @@ limitations under the License.
     <client tick="false" init="false">true</client>
     <server tick="false" init="false">true</server>
     <attribute side="server" code="0x0000" define="TEMP_MEASURED_VALUE" type="INT16S" writable="false" optional="false">measured value</attribute>
```

Comment:
This also had a `default="0x8000"` before, no?

Reply:
According to the spec there is no default value defined. The 0x8000 comes from the fact that:

> A MaxMeasuredValue of 0x8000 indicates that the maximum value is unknown.

Thats why the defaults should be 0x8000 unless the "user space application" sets them to the correct values.

Reply:
I also looked in the old version of src/app/zap-templates/zcl/data-model/silabs/ha.xml and there the default value for measured value was 0x8000 but I cannot find any text which mentions this in the matter spec. Could this be a spec issue?

Reply:
Please make sure you are reading the most updated spec.  That means after https://github.com/CHIP-Specifications/connectedhomeip-spec/pull/3224/files (which is not merged in yet).

That get rids of all mention of `0x8000`, because null is represented as TLV null in Matter, not as `0x8000`.

But more to the point, it's not clear to me whether this is trying to restore the state things were in (which had `0x8000` as default value), or trying to align with the spec, which should happen after the spec is updated to make sense (minimally after the above-linked PR is merged).

Reply:
The idea of 0x8000 was that when the accessory boots up all the clusters have a well known state. If the accessory vendor for whatever reason forgets to populate this value. At least this was my understand of 0x8000 for now.